app/controllers/twilio_controller.rb

```diff
@@ -116,7 +87,7 @@ def agent_voicemail
     status = params[:DialCallStatus] || "completed"
```

Comment:
We could add a note here about how this handles the default looping behavior of `<Gather>`'s `action` attribute.